packages/cf-component-form/src/FormLabel.js

```diff
@@ -1,24 +1,23 @@
 import React, { PropTypes } from 'react';
+import { createComponent } from 'cf-style-container';
 
-class FormLabel extends React.Component {
-  render() {
-    let className = 'cf-form__label';
+const styles = ({ theme, hidden }) => ({
+  fontSize: theme.fontSize,
+  display: hidden ? 'none' : 'block',
+  marginBottom: theme.marginBottom,
+  color: theme.color
+});
 
-    if (this.props.hidden) {
-      className += ' cf-form__label--hidden';
-    }
-
-    return (
-      <label className={className}>
-        {this.props.children}
-      </label>
-    );
-  }
-}
+const FormLabel = ({ children, className }) => (
```

Comment:
> We'll love to have a ref when interoping with other DOM manipulation libs

That sounds like something we definitely want to avoid?

> you'd think this may be faster because of some old blog post and some ancient twitter message, but it actually isn't.

It will be with Fiber.